src/xbmc_client/xbmc_client.py

```diff
@@ -68,10 +69,10 @@ def initializeXbmc(self):
             raise Exception(
                 "No host found. Have you configured the default config file %s ?"
                 % (self.getDefaultConfig()))
-        if not user:
-            raise Exception(
-                "No user found. Have you configured the default config file %s ?"
-                % (self.getDefaultConfig()))
+        # if not user:
+        #     raise Exception(
+        #         "No user found. Have you configured the default config file %s ?"
+        #         % (self.getDefaultConfig()))
```

Comment:
I believe this should not be commented for now
(I may review the file config and configuration management in the future)